internal/runners/branch/switch.go

```diff
@@ -0,0 +1,65 @@
+package branch
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/runbits"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type configurable interface {
+	CachePath() string
+}
+
+type Switch struct {
+	out     output.Outputer
+	project *project.Project
+	config  configurable
+}
+
+type SwitchParams struct {
+	Name string
+}
+
+func NewSwitch(prime primeable) *Switch {
+	return &Switch{
+		out:     prime.Output(),
+		project: prime.Project(),
+		config:  prime.Config(),
+	}
+}
+
+func (s *Switch) Run(params SwitchParams) error {
+	logging.Debug("ExecuteSwitch")
+
+	project, err := model.FetchProjectByName(s.project.Owner(), s.project.Name())
+	if err != nil {
+		return locale.WrapError(err, "err_fetch_project", s.project.Namespace().String())
```

Comment:
Third argument is fallback. Pass empty string if you don't want to use it. (This goes for any such changes in this PR).

Reply:
Right, I keep forgetting that. I think I've got them all